tests/BasicTests.elm

```diff
@@ -273,6 +286,51 @@ twoChildFormDecoded =
             }
 
 
+keyedNodeWithText : Html.Html msg
+keyedNodeWithText =
+    Keyed.ul [] [ ("0", Html.text "Hello!") ]
+
+
+keyedNodeWithTextDecoded : NodeType
+keyedNodeWithTextDecoded =
+    NodeEntry
+        { tag = "ul"
+        , children = [ TextTag { text = "Hello!" } ]
+        , descendantsCount = 1
+        , facts = emptyFacts
+        }
+
+
+keyedNodeOneChild : Html.Html msg
+keyedNodeOneChild =
+    keyedNodeWithChildren 1
+
+
+keyedNodeOneChildDecoded : NodeType
+keyedNodeOneChildDecoded =
+    keyedNodeWithChildrenDecoded 1
+
+
+keyedNodeTwoChildren : Html.Html msg
+keyedNodeTwoChildren =
+    keyedNodeWithChildren 2
+
+
+keyedNodeTwoChildrenDecoded : NodeType
+keyedNodeTwoChildrenDecoded =
+    keyedNodeWithChildrenDecoded 2
+
+
+keyedNodeOneKeyedChild : Html.Html msg
+keyedNodeOneKeyedChild =
+    keyedNodeWithKeyedChildren 1
+
+
+keyedNodeTwoKeyedChildren : Html.Html msg
+keyedNodeTwoKeyedChildren =
```

Comment:
added aliases to make the tests easier to read. let me know if you think it's overkill.